pkg/controller/common/controller_context.go

```diff
@@ -55,7 +58,16 @@ func CreateControllerContext(cb *clients.Builder, stop <-chan struct{}, targetNa
 	kubeSharedInformer := informers.NewSharedInformerFactory(kubeClient, resyncPeriod()())
 	kubeNamespacedSharedInformer := informers.NewFilteredSharedInformerFactory(kubeClient, resyncPeriod()(), targetNamespace, nil)
 	openShiftConfigKubeNamespacedSharedInformer := informers.NewFilteredSharedInformerFactory(kubeClient, resyncPeriod()(), "openshift-config", nil)
-	apiExtSharedInformer := apiextinformers.NewSharedInformerFactory(apiExtClient, resyncPeriod()())
+	// filter out CRDs that do not have the MCO label
+	filterObjectChanges := func(opts *metav1.ListOptions) {
+		ls, err := labels.ConvertSelectorToLabelsMap(opts.LabelSelector)
```

Comment:
maybe rename `ls` since it's not super clear what it is?  ConvertSelectorToLabelsMap returns Set which is a map[string]string so maybe labelsMap or labelsSet?

Reply:
Sure, can do.